services/kaptain/metadata.yaml

```diff
@@ -22,7 +22,7 @@ overview: |-
   Kaptain supports multi-tenancy with fine-grained role-based access control (RBAC) for configuring user access to volumes, as well as authentication with Dex and single sign-on with Istio.
 
   ### More information
-  - [Kaptain Documentation](https://d2iq.com/products/kaptain)
+  - [Kaptain Documentation](https://docs.d2iq.com/dkap/2.1/)
```

Comment:
Hey Max! Thank you for handling this! 
I think if we use `https://docs.d2iq.com/dkap/` instead of `https://docs.d2iq.com/dkap/2.1/` it should always take you to the latest, so we don't have to fix that for every release. But not 100% sure. 

@mwrightd2 and @Colin1841 , could you please confirm?

Reply:
That works :thumbsup: I will update it. Thanks @gvasquezvargas